openupgrade_scripts/scripts/l10n_vn/17.0.2.0.3/pre-migration.py

```diff
@@ -25,10 +25,11 @@ def _vn_coa_rename_xml_id(env):
     in order to avoid duplication
     """
     env.cr.execute(
-        """SELECT array_agg(id) FROM res_company WHERE chart_template = 'vn'"""
+        """SELECT id FROM res_company WHERE chart_template = 'vn'"""
     )
     xmlids_renames = []
-    for company_id in env.cr.fetchall():
+    company_ids = [r[0] for r in env.cr.fetchall()]
+    for company_id in company_ids:
```

Comment:
Như này mới được , để như cũ kết quả nó là for company_id in env.cr.fetchall() thì company_id sẽ là `[(1, 2)]`

Reply:
@duong77476-viindoo em để như cũ, sửa `env.cr.fetchall()` thành `env.cr.fetchall()[0]` nhé

Reply:
Phải như này mới đươc, e để như trên e cảm thấy đỡ hơn là phải 2 lần `[0][0]`
![image](https://github.com/Viindoo/OpenUpgrade/assets/56789189/87c0c98f-74f2-4a8c-afbf-5f3d6b86a866)

Reply:
ok, thanks em